common/build/eslint-config-fluid/README.md

```diff
@@ -4,9 +4,9 @@ This package contains a shared ESLint config used by all the packages in the Flu
 
 It exports the following shared ESLint configs:
 
-- `minimal`: (default) The minimal config for Fluid Framework packages.
-- `recommended`: The recommended config for Fluid Framework packages.
-- `strict`: The strictest config; intended for packages with public facing APIs.
+-   `minimal`: (default) The minimal config for Fluid Framework packages.
```

Comment:
I think it would be wise to not use prettier to reformat markdown, especially since there is auto-generated content in it. That said, we should have _something_ for markdown... maybe [remark-lint](https://github.com/remarkjs/remark-lint)?

Reply:
My main concern is tools competing with one another. For example, the packages.md file in the root is autogenerated, so I want to be sure that we don't wind up flip-flopping files depending on what tools were run in what order.

In general, though, my issues with prettier are around customization. If it's possible to configure prettier to use, say, underscores for italics and asterisks for bold, then a lot of my concerns go away.

Reply:
Ah, sorry. I completely agree with not running on generated content. I failed to note that above. I will definitely omit those generated files regardless of the decision.

I can check on the formatting options, but for the time being I'll go ahead and omit markdown and we can discuss that separately.

Reply:
Per our discussion on Teams, I've left the markdown processing in place. I also added a prettierignore file at the root, and the packages that didn't need to specify local overrides are now using that.